### DISCUSSION TITLE
On back press handler like on Android

### DISCUSSION THREAD

**Discussion by N7ghtm4r3** [Tlaster/PreCompose#376] [Ideas]
Hi, I tried to use `BackHandler` but I don't know if is the handler for the `onBack` action like `onBackPress` method like on Android.
If it isn't that method is possible to integrate one?

**Comment by Tlaster** [comment]
`moe.tlaster.precompose.navigation.BackHandler` uses `androidx.activity.compose.LocalOnBackPressedDispatcherOwner` under the hook, so it's identical with the `androidx.activity.compose.BackHandler`

**Reply by N7ghtm4r3** [reply]
Thank you!
